### PR TITLE
Remove duplicated code for picking the default jar path

### PR DESCRIPTION
**Pull request by manuelciosici**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,16 +2,16 @@
 
 from setuptools import setup
 
-with open(os.path.join("scienceworld", "version.py")) as f:
-    VERSION = f.readlines()[0].split("=")[-1].strip("' \n")
+from scienceworld import scienceworld
```
Comment (aphedges) [inline setup.py:5]: I'm getting the error `AttributeError: module 'scienceworld.scienceworld' has no attribute 'version'` later on, so you might fix it by trying the following:

```suggestion
import scienceworld
```

**Review by aphedges — CHANGES_REQUESTED** [review]
I suggest you test this code some more. Running `pip install --force-reinstall --no-deps -U 'scienceworld @ git+https://github.com/manuelciosici/ScienceWorld.git@remove_duplicate_jar_path'` fails to build the wheel.

The reason `version.py` is parsed instead of imported is due to peculiarities in the Python build system. The approach is pretty common, and it's even done automatically for some configurations of `setuptools`.

File: setup.py (hunk 1 above)
Comment (manuelciosici) [reply setup.py:5]: @aphedges Good catch. I'll test some more. It worked for me yesterday, but maybe I messed something up. I probably had `scienceworld` in `PYTHONPATH`.

Comment (MarcCote) [reply setup.py:5]: I'm not sure how this would behave if you already have a version of `scienceworld` installed (or residing on your PYTHONPATH).

**Review by MarcCote — CHANGES_REQUESTED** [review]
I like the idea of putting the JAR_PATH to a util file, but I wouldn't import scienceworld in setup.py.

**Comment by manuelciosici** [comment]
I think @aphedges is right about this not working. I must have messed up something with my import path. I think we should close this PR and ignore it.

**Pull request closed without merging**